framework/CodeInterfaces/CobraTF/ctfdata.py

```diff
@@ -18,24 +18,34 @@ class ctfdata:
     Class that parses CTF output file and reads in (output files type: .ctf.out) and write a csv file
   """
 
-  def __init__(self, filen):
+  # def __init__(self, filen):
+  def __init__(self,filen):
+
     """
       Constructor
       @ In, filen, string, file name to be parsed
       @ Out, None
     """
+
     # check file existence (1)
-    if "ctf.out" not in filen:
+    if ("ctf.out" not in filen) and  ("deck.out" not in filen):
```

Comment:
deck.out is an hardcoded name of Cobra?

Reply:
Actually yes, but I removed that part. It is either ‘inputName.ctf.out’ or  ‘deck.out’ 

Currently I removed the deck.out option